CMakeLists.txt

```diff
@@ -130,6 +131,11 @@ if(CGAL_FOUND)
     message(STATUS "  Libraries : ${CGAL_LIBRARY}")
 endif()
 
+if (TORCH_ENABLED)
+    set(CMAKE_TORCHLIB_PATH ${CMAKE_CURRENT_SOURCE_DIR}/lib/libtorch)
```

Comment:
The CMake config is based on existence of LibTorch as part of COLMAP lib. Did not include in the PR as this should likely change and require it as an external dependency